lms/djangoapps/teams/static/teams/js/views/topics.js

```diff
@@ -21,11 +22,18 @@
                 PaginatedView.prototype.initialize.call(this);
             },
 
+            createHeaderView: function () {
+                return new PagingHeader({
+                    collection: this.options.collection,
+                    srInfo: this.srInfo,
+                    showSortControls: true
+                });
+            },
+
             render: function() {
                 var self = this;
                 this.collection.refresh()
                     .done(function() {
-                        self.collection.isStale = false;
                         PaginatedView.prototype.render.call(self);
```

Comment:
I realize this wasn't added in this PR, but I'm pretty sure that this line is unnecessary if the header, list, and footer views properly listen to their collections/models.